6.0/6.0.0a4/Dockerfile.python39

```diff
@@ -0,0 +1,58 @@
+FROM python:3.9-slim-bullseye as base
+FROM base as builder
+
+ENV PIP_PARAMS="--use-deprecated legacy-resolver"
```

Comment:
The legacy resolver is not needed anymore with the pip 22 that we are pinning. (And for clarity: we can use pip 22 because we use latest zc.buildout.)

Reply:
Done. For the moment I set the vars to "", just in case we need them again at some point.